api/tests/test_hyp3_api.py

```diff
@@ -71,28 +73,90 @@ def test_submit_job(client, states_stub):
     }
 
 
+@mock_dynamodb2
 def test_list_jobs(client):
+    table_name = environ['TABLE_NAME']
+    hyp3_api.DYNAMODB_RESOURCE = boto3.resource('dynamodb')
+
+    table = hyp3_api.DYNAMODB_RESOURCE.create_table(
+        TableName=table_name,
+        KeySchema=[
+            {
+                'AttributeName': 'job_id',
+                'KeyType': 'HASH'
+            },
+        ],
+        GlobalSecondaryIndexes=[
+            {
+                'IndexName': 'user_id',
+                'KeySchema': [
+                    {
+                        'AttributeName': 'user_id',
+                        'KeyType': 'HASH',
+                    },
+                ],
+                'Projection': {
+                    'ProjectionType': 'ALL'
+                }
+            }
+        ],
+        AttributeDefinitions=[
+            {
+                'AttributeName': 'job_id',
+                'AttributeType': 'S'
+            },
+            {
+                'AttributeName': 'user_id',
+                'AttributeType': 'S'
+            },
+
+        ],
+        ProvisionedThroughput={
+            'ReadCapacityUnits': 15,
+            'WriteCapacityUnits': 15
+        }
+    )
+
+    items = [
+        {
+            'job_id': '0ddaeb98-7636-494d-9496-03ea4a7df266',
+            'user_id': 'user_with_jobs',
+            'parameters': {
+                'granule': 'S1A_IW_GRDH_1SDV_20200426T125708_20200426T125733_032299_03BCC4_b4E0'
```

Comment:
we'll need a comma here eventually but that's really pedantic of me, it's fine as is